Azure-Samples/modern-data-warehouse-dataops
# [Fabric E2E Sample] Add script to upload config file and seed data

# Type of PR
- Code changes

## Purpose
- The PR is to add the part of IaC of uploading the config file and seed data in to sc-adls-main:
        - Files/sc-adls-main/config/lakehouse_ddls.yaml 
        - Files/sc-adls-main/config/application.cfg (derived using application.cfg.template during ci/cd process). 
        - Files/sc-adls-main/reference/dim_date.csv
        - Files/sc-adls-main/reference/dim_time.csv  

## Does this introduce a breaking change? If yes, details on what can break
No.

## Author pre-publish checklist


## Validation steps
- Run the ./deploy.sh and check the following files in the storage account and container:
  - config/lakehouse_ddls.yaml 
  - config/application.cfg
  - reference/dim_date.csv
  - reference/dim_time.csv  

## Issues Closed or Referenced
- #798 
- #870 
- #974
